app/src/main/java/com/celzero/bravedns/service/PersistentState.kt

```diff
@@ -284,9 +279,9 @@ class PersistentState {
 
 
         fun setFirewallModeForScreenState(context : Context , state : Boolean) {
-            val editor: SharedPreferences.Editor = getUserPreferences(context).edit()
-            editor.putBoolean(SCREEN_STATE, state)
-            editor.apply()
+            getUserPreferences(context).edit {
+                putBoolean(SCREEN_STATE, state)
```

Comment:
is the default `#apply` or `#commit`?

Reply:
The default is apply
![image](https://user-images.githubusercontent.com/25826502/103090961-e4aa1800-4607-11eb-85d5-7b51f92ee025.png)